src/utils/processDescription/handlers/index.js

```diff
@@ -150,7 +150,6 @@ const memberDB = new Set([
   { name: 'pfam', re: /^PF[0-9]{5}$/i },
   { name: 'pirsf', re: /^PIRSF[0-9]{6}$/i },
   { name: 'prints', re: /^PR[0-9]{5}$/i },
-  { name: 'prodom', re: /^PD[A-Z0-9]{6}$/i },
```

Comment:
Another candidate for moving into MySQL? The thinking being that it keeps all secondary data related to addition or removal of member (or other) databases in one place.